pkg/driver/videotest/dummy.go

```diff
@@ -144,3 +152,7 @@ func (d dummy) Properties() []prop.Media {
 		},
 	}
 }
+
+func (d *dummy) Status() driver.State {
+	return d.state
```

Comment:
should these states be atomic values?

Reply:
It wasn't before so I didn't change it to be. The `adapterWrapper` class just returns and modifies its values without any mutexes so pushing down this logic won't cause any new issues. It seems like the `adapterWrapper` class in general should be protected by mutexes. I think that would be out of scope for this change though.